src/system/data-transfer-tax-rates.md

```diff
@@ -62,15 +65,15 @@ You now have two spreadsheets open, side by side. One with the Magento export fi
 
 1. Save the prepared data as a .CSV file.
 
-   When prompted, verify that a comma is used a **Field delimiter** and double quotes as the **Text delimiter** and click **OK**.
+   When prompted, verify that a comma is used a Field delimiter and double quotes as the Text delimiter. Then click **OK**.
```

Comment:
```suggestion
   When prompted, verify that a comma is used as the Field delimiter and double quotes as the Text delimiter. Then click **OK**.
```

Reply:
made this correction as part of Erik's review